ca/ca_test.go

```diff
@@ -338,9 +338,11 @@ func TestIssuePrecertificate(t *testing.T) {
 		// |testCases| because the "certificate-for-precertificate" tests use
 		// the precertificates previously generated from the preceding
 		// "precertificate" test.
+		testCase := testCase
```

Comment:
Do this before the comment. Here, it is separating the comment from the code it is commenting.